include/deal.II/lac/la_vector.templates.h

```diff
@@ -26,6 +26,51 @@ DEAL_II_NAMESPACE_OPEN
 namespace LinearAlgebra
 {
   template <typename Number>
+  void Vector<Number>::reinit(const size_type size,
+                              const bool omit_zeroing_entries)
+  {
+    ReadWriteVector<Number>::reinit(size, omit_zeroing_entries);
```

Comment:
Ah, so it is to sort out the issues with the multiple inheritance and the question on which of the `reinit` methods should be present given that both classes have `reinit` methods (but with different signatures). Now I understand, thanks. (I'm always a bit unsure about the C++ rules with multiple inheritance and which of the methods are actually visible.)